materialize-sql/.snapshots/TestTableTemplate

```diff
@@ -1,10 +1,10 @@
 
 	CREATE TABLE one."reserved".checkpoints (
-			materialization TEXT,
-			key_begin BIGINT,
-			key_end BIGINT,
-			fence BIGINT,
-			checkpoint TEXT,
+			materialization TEXT NOT NULL,
```

Comment:
These snapshot updates are not a result of anything in this PR, they just haven't been updated since we went back to creating columns as `NOT NULL` if they are required in collection schemas. I've added running the `materialize-sql` package tests to SQL materialization connectors so we don't get out of sync in the future.